Runtime/Model/Database/BacktraceDatabaseSettings.cs

```diff
@@ -18,6 +19,7 @@ public BacktraceDatabaseSettings(string databasePath, BacktraceConfiguration con
 
             DatabasePath = databasePath;
             _configuration = configuration;
+            _retryInterval = configuration.RetryInterval >= 0 ? Convert.ToUInt32(_configuration.RetryInterval) : 60;
```

Comment:
Why not use a direct cast? Also, you did not use the constant defined in `BacktraceConfiguration`.

```csharp
_retryInterval = (uint)(configuration.RetryInterval >= 0 ? _configuration.RetryInterval : BacktraceConfiguration.DefaultRetryInterval);
```

Reply:
I forgot about constant ! 
Let me also check (unit) cast in old .net platforms. For some reasons we decided to use convert instead.